helix-core/src/test/java/org/apache/helix/integration/TestBatchEnableInstances.java

```diff
@@ -49,7 +43,7 @@ public void beforeClass() throws Exception {
     _accessor = new ConfigAccessor(_gZkClient);
   }
 
-  @Test
+  @Test(enabled = false)
```

Comment:
May be it is just me, but i feel we should keep the test and make sure we get 'NotImplemented' exception. This way we know for sure that if anyone attempts to use batch APIs,  will get NotImplemented.

Reply:
Good point. I found it hard to enable and modify existing tests, so I add a new test for this coverage. Thanks for the input.